src/CoderDojo/WebsiteBundle/Form/Type/CocRequestFormType.php

```diff
@@ -50,13 +50,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'Achternaam'
             ))
             ->add('email', EmailType::class, [
-                'label' => 'Email'
+                'label' => 'Email',
+                'attr' => [
+                    'placeholder' => 'Let op, dit moet een persoonlijk emailadres zijn.'
```

Comment:
Aaah, then I'd add a notion that the `@coderdojo-[city].nl` emails are not to be used too. I just submitted a request to all our mentors for their details but planned on using their dojo mails. (Actually already submitted one like that)